src/servers/src/opentsdb/codec.rs

```diff
@@ -126,6 +126,7 @@ impl DataPoint {
     }
 
     pub fn as_grpc_insert(&self) -> GrpcInsertRequest {
+        let catalog_name = DEFAULT_CATALOG_NAME.to_string();
         let schema_name = DEFAULT_SCHEMA_NAME.to_string();
```

Comment:
Is it safe to use defaults here?

Reply:
I will refactor opentsdb,prometheus etc in next pr.